src/styles/components/_audiovideo.scss

```diff
@@ -71,6 +71,14 @@
   }
 }
 
+.mainPresentation #audiovideo.fullscreen {
+  top: 51px !important;
+}
+
+.mainYoutubevideo #audiovideo.fullscreen {
+  top: 51px !important;
```

Comment:
Due to [this declaration](https://github.com/strukturag/spreed-webrtc/blob/develop/src/styles/components/_audiovideo.scss#L62) it is required to set an offset from the top.